### PR TITLE
Add target to build the kvm driver in docker

### PR DESCRIPTION
**Pull request by afbjorklund**
Helps with portability, by building against older OS.

This is for #3689

### CODE REVIEW

**Comment by afbjorklund** [comment]
Mostly something to have ready for 1.0 - seems like Travis wants a copyrighted Dockefile

**Comment by tstromberg** [comment]
You should be able to automatically get boilerplate by running `./hack/boilerplate/fix.sh`

File: installers/linux/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:16.04
```
Comment (sharifelgamal) [inline installers/linux/Dockerfile]: Could you switch this to using `gcr.io/gcp-runtimes/ubuntu_16_0_4`? That way we don't have rely on dockerhub being to release kvm.

Comment (afbjorklund) [reply installers/linux/Dockerfile]: Ok

**Comment by afbjorklund** [comment]
Redid the Dockerfile (still for Ubuntu 16.04), based on the official golang (which uses Debian Stretch).

**Pull request merged**